src/main/java/com/neva/felix/webconsole/plugins/search/decompiler/jd/Buffer.java

```diff
@@ -0,0 +1,385 @@
+package com.neva.felix.webconsole.plugins.search.decompiler.jd;
+
+import java.nio.BufferUnderflowException;
+import java.util.Arrays;
+
+/**
+ * @author Mike Strobel
+ */
+public class Buffer {
```

Comment:
it would be nice to keep info about source of that code / link to origin repo etc

Reply:
True I'll add it to both classes, the other also came from the old compiler.